lcls-twincat-math/_Config/PLC/lcls_twincat_math.xti

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0"?>
+<TcSmItem xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://www.beckhoff.com/schemas/2012/07/TcSmProject" TcSmVersion="1.0" TcVersion="3.1.4024.35" ClassName="CNestedPlcProjDef">
+	<Project GUID="{1849AE5F-FA91-4B6D-8736-73C8239AD67D}" Name="lcls_twincat_math" PrjFilePath="..\..\lcls_twincat_math\lcls_twincat_math.plcproj" TmcFilePath="..\..\lcls_twincat_math\lcls_twincat_math.tmc" ReloadTmc="true" AmsPort="851" FileArchiveSettings="#x000e" SymbolicMapping="true">
+		<Instance File="lcls_twincat_math Instance.xti"/>
```

Comment:
This is the instance link that confuses pytmc
TODO for me is to investigate how to make this work properly or understand how this file gets created

Reply:
Issue created on pytmc repository.